oar/core/advisory.py

```diff
@@ -749,3 +751,45 @@ def get_overall_grade(self):
         else:
             raise AdvisoryException(
                 f"error when accessing advisory containers - {self.errata_id}")
+
+    def check_kernel_tag(self):
+        # get advisories url
+        """
+        Get kernel tag from advisory build image.
+
+        Returns:
+            bool: True if kernel tagged with early-kernel-stop-ship.
+        """
+        candidate_builds = self._cs.get_candidate_builds()
+        if candidate_builds:
+            for k, v in candidate_builds.items():
+                build_url = "https://errata.devel.redhat.com/api/v1/erratum/"+v+"/builds"
+                logger.info(k+" build url:"+v+"\n")
+                session = requests.Session()
+                session.hooks['response'] = lambda response, *args, **kwargs: response
+                session.auth = ('', '')
+                kerberos_auth = HTTPKerberosAuth(mutual_authentication=REQUIRED)
+                try:
+                    response = session.get(build_url, auth=kerberos_auth)
+                    if response.status_code == 401:
+                        logger.error("401 Unauthorized: Authentication failed or missing.")
+                    else:
+                        rhos_nvr = jq.compile('.. | objects | to_entries[] | select(.key | startswith("rhcos-x86")) | .key').\
+                            input(text=response.text).all()
+                        logger.info("rhcos nvr is:"+rhos_nvr[0])
+                except BaseException as e:
```

Comment:
when using` jq.compile`, you should mainly be prepared to catch the` jq.SyntaxError` exception to handle cases where the jq filter has incorrect syntax

Reply:
fixed and committed